flink-core/src/test/java/org/apache/flink/api/java/typeutils/runtime/PojoSerializerUpgradeTest.java

```diff
@@ -40,7 +42,15 @@ public PojoSerializerUpgradeTest(TestSpecification<Object, Object> testSpecifica
 	@Parameterized.Parameters(name = "Test Specification = {0}")
 	public static Collection<TestSpecification<?, ?>> testSpecifications() throws Exception {
 		ArrayList<TestSpecification<?, ?>> testSpecifications = new ArrayList<>();
-		for (MigrationVersion migrationVersion : MIGRATION_VERSIONS) {
+		// for PojoSerializer we also test against 1.7 and newer because we have snapshots
+		// for this which go beyond what we have for the usual subclasses of
+		// TypeSerializerUpgradeTestBase
+		List<MigrationVersion> testVersions = new ArrayList<>();
+		testVersions.add(MigrationVersion.v1_7);
+		testVersions.add(MigrationVersion.v1_8);
+		testVersions.add(MigrationVersion.v1_9);
+		testVersions.addAll(Arrays.asList(MIGRATION_VERSIONS));
```

Comment:
IICU, we do not support v1_10 here because of there did not have the snapshot for 1_10, do we need to add a comment for this.

Reply:
You're right, I'm extending the comment here.